source/agora/common/DNS.d

```diff
@@ -399,6 +399,16 @@ public struct Header
     /// Reserved for future use. Must be zero in all queries and responses.
     public ubyte Z () const scope { return (this.field2 & 0b0_0000_0_0_0_0_111_0000) >> 4; }
 
+    /// Authenticated Data, initially defined in RFC2065
+    public bool AD () const scope { return !!(this.field2 & (1 << 5)); }
+    /// Ditto
+    public void AD (bool val) scope { this.field2 = (this.field2 & ~(1 << 5)) | (1 << 5); }
```

Comment:
`val` should be used instead of directly `1`
`this.field2 = (this.field2 & ~(1 << 5)) | (val << 5);`

Reply:
Riiiiiight... Fixed!